dashboard/app/models/user.rb

```diff
@@ -1629,11 +1629,16 @@ def should_see_inline_answer?(script_level)
       (script_level && UserLevel.find_by(user: self, level: script_level.level).try(:readonly_answers))
   end
 
+  # rubocop:disable Lint/UnreachableCode
   def show_census_teacher_banner?
+    # Temporarily hide the banner
```

Comment:
How temporary? And will we want to be able to override hide/show like this in the future?  If so, we'll probably want a more robust solution than disabling rubcop and returning false. If temporary means a couple days and we don't anticipate needing to do this again, then I guess it seems fine.

Reply:
The most obvious immediate alternative would be to simply replace the function with `false` but I prefer keeping code intact that we know we're about to revive.